src/block.ml

```diff
@@ -17,7 +17,7 @@ module Pre = struct
     | Rblockquote of t
     | Rlist of Block_list.kind * Block_list.style * bool * int * block list list * t
     | Rparagraph of string list
-    | Rfenced_code of int * int * Code_block.kind * (string * string) * string list
+    | Rfenced_code of int * int * Code_block.kind * (string * string) * string list * attributes
```

Comment:
It may be a good idea to use inline records here, but we can leave this for later.

Reply:
I'll try to do that on another PR

Reply:
Sure, but don't worry about it too much, it is not urgent :)